src/image/ImageAnnotator.js

```diff
@@ -37,10 +37,7 @@ class ImageAnnotator extends Annotator {
         let location = null;
 
         let clientEvent = event;
-        if (this.isMobile) {
-            if (!event.targetTouches || event.targetTouches.length === 0) {
-                return location;
-            }
+        if (this.isMobile && event.targetTouches && event.targetTouches.length > 0) {
```

Comment:
Were you seeing touch events without any targetTouches? I don't think mobile Chrome provides that, I think it's a just a `touchList` array?

Reply:
So I'm actually unsure which is the better option here. So turns out that when this method is called from [controller.getIntersectingThreads()](https://github.com/box/box-annotations/blob/master/src/controllers/AnnotationModeController.js#L459), the mouseEvent may not always have targetTouches on mobile devices. So there is a place where we are not guaranteed to have targetTouches. 

I am not sure which made sense 
1) This method of continuing to use the passed in event if there are no targetTouches on mobile devices 
2) Modifying the controller.getIntersectingThreads() method to fake targetTouches on mobile devices i.e.
```
const clientEvent = event;
if (this.isMobile && (!clientEvent.targetTouches || clientEvent.targetTouches.length === 0)) {
    // Fake targetTouches array with current mouse event
    clientEvent.targetTouches = [ event ];
}
const location = this.annotator.getLocationFromEvent(clientEvent, TYPES.point);
```

Reply:
@jeremypress @pramodsum  `targetTouches` should be present if the touch began on the same element as it ends. Its type is `TouchList`. As far as I can remember, we're checking this to prevent users from highlighting text, and moving the touch to outside of the preview viewport and selecting other pieces of text AND a form of cancelling the highlight if the user leaves the viewport.

Instead of checking `isMobile`, why not just refer to `hasTouch`? That might do away with issues related to stylus registering as clicks, not touches.

Reply:
So it seems like since this is causing issues because for highlights we use mouseup rather than click to determine whether or not we want to show the respective dialog. I can leave the image related portion as is and then keeping my change for highlights (still swapping out hasTouch for isMobile for both options)